### ISSUE TITLE
name@slugifiedName

### ISSUE THREAD

**Issue by mnot** [rfc-format/draft-iab-xml2rfc-v3-bis#166]
In the 2.32 revision of the RNC, the `slugifiedName` attribute was added to the `name` element. This should be discussed and documented.

**Comment by jrlevine** [comment]
Derived attribute, should not occur in a submitted draft.

**Comment by stpeter** [comment]
Agreed, this is a derived attribute generated by the preptool and shouldn’t be used by authors. We need to update the documentation to define the attribute.
